environs/broker.go

```diff
@@ -105,6 +105,11 @@ type StartInstanceParams struct {
 // StartInstanceResult holds the result of an
 // InstanceBroker.StartInstance method call.
 type StartInstanceResult struct {
+	// DisplayName is an optional human-readable string that's used
+	// for display purposes only. When empty, the output defaults to
```

Comment:
No need for the 2nd sentence - it's up to individual renderers how this is used and not really the concern of this layer